src/net_processing.cpp

```diff
@@ -2366,7 +2366,7 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
         // Each connection can only send one version message
         if (pfrom.nVersion != 0)
         {
```

Comment:
Only if you retouch the branch:

```suggestion
        if (pfrom.nVersion != 0) {
            // Each connection can only send one version message
```

Reply:
My preference would be to remove the comment, because the logprint is already clear enough

Reply:
Thx, fixed.